frontend/plan/components/selector/TagList.tsx

```diff
@@ -1,11 +1,22 @@
 import React, { useState } from "react";
+import styled from "styled-components";
 import Tag from "./Tag";
 
 interface TagListProps {
     elements: string[];
     limit?: number;
     onClick?: (id: string) => void;
 }
+
+const HiddenTagListContainer = styled.span`
+    overflow: hidden;
+    display: inline-block;
+    height: ${({ expanded }: { expanded: boolean }) => (expanded ? "auto" : 0)};
```

Comment:
Ditto on the interface here. Relevant docs: https://styled-components.com/docs/api#using-custom-props